mdal/mdal_utils.cpp

```diff
@@ -1112,13 +1112,17 @@ MDAL::Library::Library( std::string libraryFile )
 MDAL::Library::~Library()
 {
   d->mRef--;
+  if ( d->mRef == 0 )
```

Comment:
I think it would be slightly more readable like:
```cpp
  if ( d->mRef == 0 )
  {
    if ( d->mLibrary )
    {
#ifdef _WIN32
      FreeLibrary( d->mLibrary );
#else
      dlclose( d->mLibrary );
#endif
    }
    delete d;
  }
```

Reply:
Ok, done